src/components/organisms/AccountPage/ProfileTab/FormFields/index.tsx

```diff
@@ -121,7 +147,11 @@ export function FormFields() {
           defaultValue={mentor.data?.aboutMe}
         />
 
-        <SectionLegend>Máximo 600 caracteres.</SectionLegend>
+        <CharacterSectionLegend
+          className={`${isMaxCharactersExceeded === true ? 'error' : ''}`}
```

Comment:
se `isMaxCharactersExceeded` for sempre boolean, que acredito que seja, vc não precisa do === true, só fazer `isMaxCharactersExceeded ?  'error' : ''`

Reply:
entendi, fiz correção